week-8/Homework/mandatory/3-dog-photo-gallery/dog.js

```diff
@@ -0,0 +1,37 @@
+function getData(URL) {
+  fetch(URL)
+    .then((response) => {
+      return response.json();
+    })
+    .then((json) => {
+      dataBase(json);
+    })
+    .catch((err) => console.log(err));
+}
+
+function dataBase(message) {
+  addPicButton(message);
+}
+
+let ulList = document.getElementById(`ul`);
```

Comment:
Was there a reason to declare this ulList outside of the `dogyPic` function?

Reply:
oh no...
I just forgot to put it back,
to have smaller functions I decided to make two functions and Bothe needed that `ulList`, but finally I regret and forgot to put that back.